cmd/create/cluster/cmd.go

```diff
@@ -653,23 +658,38 @@ func init() {
 	)
 	flags.MarkHidden("billing-account")
 
+	flags.BoolVar(
+		&args.createAdminUser,
+		"create-admin-user",
+		false,
+		`Create cluster admin`,
+	)
+
+	flags.StringVar(
+		&args.createAdminPassword,
+		"create-admin-password",
```

Comment:
I'm not sure about the flag naming convention.
With `create-admin-password` one can expect this flag to be a boolean.
I wonder if we should go for something like `cluster-admin-password`, WDYT?

Reply:
Hi @oriAdler
The changes requested here definitely seems more intuitive. I will go ahead and make this change.
In addition, since it will now be `create-admin-user` and `cluster-admin-password`, should we enforce flag `cluster-admin-password` to always be used with `create-admin-user`? or should we allow users to be able to 
use  `cluster-admin-password` without `create-admin-user`?

Related to confusion here: https://github.com/openshift/rosa/pull/1522#discussion_r1341568029

Reply:
IMO only if the user assigns a password.
So if the user selected an auto-generated password option, we don't print `cluster-admin-password`, and the next time the user runs the command the password will be auto-generated.
With that being said, I think both options apply.

Reply:
the output of the build command after the commit will show as 
`rosa create cluster --create-admin-user` if the user decides to create admin user without custom password.
`rosa create cluster --create-admin-user` if the user decides to create admin user with custom password.